Important Note to a resource where are all the options #5072 (#5098)
MicrosoftDocs/windows-itpro-docs: devices/surface/surface-dock-firmware-update.md

For more information, refer to [Command line options](https://docs.microsoft.com/windows/win32/msi/command-line-options) documentation.

> [!IMPORTANT]

> > Also, if you want to keep your surface dock updated using any other method, please [follow the link](https://support.microsoft.com/help/4023478/surface-update-your-surface-dock).

Reviewer: ```suggestion
> If you want to keep your Surface Dock updated using any other method, please see [Update your Surface Dock](https://support.microsoft.com/help/4023478/surface-update-your-surface-dock) for details.
```
